### ISSUE TITLE
Add flag for continuous color

### ISSUE THREAD

**Issue by behinger** [piermorel/gramm#1]
currently continuous color is activated when there are more than 15 categories.

It would be nice to be able to manually activate continuous colors. In earlier versions you could manually set  g.continuous_color=1 (albeit with some bugs in plotting).

**Comment by piermorel** [comment]
I restricted access to private variables in the latest update for simplicity, it should be simple to allow setting this property again. What kind of plotting bugs were you observing before ? I might as well try to fix them while I'm at it.

**Comment by behinger** [comment]
![screenshot from 2016-01-29 11 47 48](https://cloud.githubusercontent.com/assets/10183650/12673625/26eb1bf6-c67e-11e5-97b1-f1d3c3528668.png)
Something like this (this is now with the current gramm version). The endpoints are still connected to the starting points.
Do you need a minimal working example?

**Comment by piermorel** [comment]
I added the set_continuous_color() function that can be used to force continuous colors, and I also fixed some bugs with the start and end points of continuous color lines... Producing these lines is a massive hack anyway (as there is the possibility to vary the color within a line), but it should behave much better now. If you still have connecting line problems, it could just be about telling gramm which points belong to which lines, which it can't guess. To fix that, which should be easy enough, check the "Example of all the different input formats for x and y" section of the examples.m file.
